extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -563,13 +565,31 @@ describes.realWin('amp-ad-network-adsense-impl', {
         }));
     });
 
-    it('should return true if in experiment', () => {
-      forceExperimentBranch(impl.win, ADSENSE_A4A_EXPERIMENT_NAME,
-          ADSENSE_EXPERIMENT_FEATURE.DELAYED_REQUEST);
-      expect(impl.delayAdRequestEnabled()).to.be.true;
+    Object.entries({
+      [ADSENSE_EXPERIMENT_FEATURE.DELAYED_REQUEST_EXTERNAL_CONTROL]: {
+        layer: ADSENSE_A4A_EXPERIMENT_NAME,
+        result: false,
+      },
+      [ADSENSE_EXPERIMENT_FEATURE.DELAYED_REQUEST_EXTERNAL]: {
+        layer: ADSENSE_A4A_EXPERIMENT_NAME,
+        result: true,
+      },
+      [INTERNAL_FAST_FETCH_DELAY_REQUEST_EXP.CONTROL]: {
+        layer: FF_DR_EXP_NAME,
+        result: false,
+      },
+      [INTERNAL_FAST_FETCH_DELAY_REQUEST_EXP.EXPERIMENT]: {
+        layer: FF_DR_EXP_NAME,
+        result: true,
+      },
+    }).forEach(item => {
+      it(`should return ${item[1].result} if in ${item[0]} experiment`, () => {
+        forceExperimentBranch(impl.win, item[1].layer, item[0]);
+        expect(impl.delayAdRequestEnabled()).to.equal(item[1].result);
```

Comment:
This test isn't really any different than the other one that just directly tests fastFetchDelayedRequestEnabled. If you consolidate the function you'll only need one of these, and it won't be so redundant.